### PR TITLE
ansible: add authentication

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

File: contrib/ansible/roles/skydive_common/tasks/config.yml [hunk 1]
```diff
@@ -26,6 +26,69 @@
...
+    edits:
+        - key: "auth.keystone.auth_url"
+          value: "{{ os_auth_url }}"
+        - key: "auth.keystone.tenant_name"
+          value: "{{ os_tenant_name }}"
+        - key: "auth.analyzer_username"
+          value: "{{ os_username }}"
+        - key: "auth.analyzer_password"
+          value: "{{ os_password }}"
+        - key: "auth.keyston.tenant_name"
```
Comment (nplanel) [inline contrib/ansible/roles/skydive_common/tasks/config.yml]: should be removed   key: "auth.keyston.tenant_name" as it's misspell and you already set auth.keystone.tenant_name

File: contrib/ansible/roles/skydive_agent/tasks/docker.yml [hunk 2]
```diff
@@ -27,6 +27,6 @@
         -e SKYDIVE_NETNS_RUN_PATH=/host/run {{ skydive_agent_docker_extra_env }} \
         -p {{ skydive_agent_port }}:{{ skydive_agent_port }} \
         --name=skydive-agent-{{ ansible_hostname }} \
-        {{ skydive_docker_registry }}/{{ skydive_agent_docker_image }}:{{ skydive_docker_image_tag }} \
+        {{ skydive_docker_registry }}/{{ skydive_agent_docker_image }}:{{ skydive_agent_docker_image_tag }} \
```
Comment (nplanel) [inline contrib/ansible/roles/skydive_agent/tasks/docker.yml]: why having/using agent and analyzer different docker tag ?

**Comment by safchain** [comment]
run vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun selenium-tests

File: contrib/ansible/roles/skydive_agent/tasks/config.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  yedit:
+    src: "{{ skydive_config_file }}"
+    key: "agent.topology.neutron"
+    state: absent
+
+- name: Setup neutron probe section
+  yedit:
+    src: "{{ skydive_config_file }}"
+    edits:
+        - key: "agent.topology.neutron.auth_url"
```
Comment (nplanel) [inline contrib/ansible/roles/skydive_agent/tasks/config.yml]: openstack.auth_url
:)

Comment (safchain) [reply contrib/ansible/roles/skydive_agent/tasks/config.yml]: not yet as the stable version is not using the new params, need to wait a bit before  changing here as well

Comment (safchain) [reply contrib/ansible/roles/skydive_agent/tasks/config.yml]: I mean packages, etc...

**Comment by lebauce** [comment]
contrib/ansible/inventory/hosts.locahost => contrib/ansible/inventory/hosts.localhost
(same for all the other files)

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun selenium-tests

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by nplanel** [comment]
run skydive-vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Comment by safchain** [comment]
rerun vagrant-tests

**Pull request merged**